core/spec/models/spree/adjustment_spec.rb

```diff
@@ -13,6 +13,32 @@
 
   let(:adjustment) { Spree::Adjustment.create!(label: 'Adjustment', adjustable: order, order: order, amount: 5) }
 
+  describe 'scopes' do
```

Comment:
@nishant-cyro So maybe we could add a ```context``` block for completion scopes? But it's just an idea, it seems fine to me now.